tests/test_downstream_issue.py

```diff
@@ -1293,21 +1293,24 @@ def test_matching_jira_issue_query(
         mock_check_comments_for_duplicates.return_value = True
         mock_find_username.return_value = "mock_username"
 
-        # Call the function
-        response = d._matching_jira_issue_query(
-            client=mock_client, issue=self.mock_issue, config=self.mock_config
-        )
+        # Mock the Snowflake query function
+        with mock.patch(
+            "sync2jira.downstream_issue.execute_snowflake_query"
+        ) as mock_snowflake:
```

Comment:
Given that this test function (as nearly all of ours do) uses a function-level decorator to install function mocks, is there a good reason to use this alternate approach for `execute_snowflake_query()`?  If not, it would be better to be consistent and use a decorator to apply the patch and then set up the mock as we do the others just above this code.